docs/source/using/addons.adoc

```diff
@@ -180,6 +180,21 @@ Multiple variables need to be comma separated.
 # Required-Vars: PROJECT_USER
 ----
 
+Default values for variables can be achieved using the _Var-Defaults_ metadata header.
```

Comment:
"achieved" does not really fit in this context. Maybe something like:

You can set a default value for a variable using the _Var-Defaults_ metadata header.